recirq/quantum_chess/controlled_iswap.py

```diff
@@ -116,8 +117,8 @@ def simplify_op(op: cirq.Operation) -> cirq.Operation:
     )
 
     converted_ops = cirq.Circuit(convert_op(op) for op in circuit.all_operations())
-    cirq.optimizers.MergeSingleQubitGates().optimize_circuit(converted_ops)
-    cirq.optimizers.DropNegligible().optimize_circuit(converted_ops)
+    optimizers.MergeSingleQubitGates().optimize_circuit(converted_ops)
```

Comment:
Actually, can you remove the import above and change this to:

`cirq.MergeSingleQubitGates().optimize_circuit(converted_ops)`

This is safer, since we (cirq-maintainers) don't guarantee that package names will never change.  If we move this class around, the top-level import will still work.